targets/microchip-icicle-kit.nix

```diff
@@ -0,0 +1,76 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Polarfire Enablement Kit
+{
+  self,
+  lib,
+  nixpkgs,
+  nixos-hardware,
+}: let
+  name = "microchip-icicle-kit";
+  system = "riscv64-linux";
+  microchip-icicle-kit = variant: extraModules: let
+    hostConfiguration = lib.nixosSystem {
+      inherit system;
+      specialArgs = {inherit lib;};
+      modules =
+        [
+          nixos-hardware.nixosModules.microchip-icicle-kit
+          ../modules/hardware/polarfire/mpfs-nixos-sdimage.nix
+          ../modules/host
+
+          {
+            appstream.enable = false;
+            boot = {
+              enableContainers = false;
+              loader = {
+                grub.enable = false;
+                generic-extlinux-compatible.enable = true;
+              };
+            };
+
+            # Disable all the default UI applications
+            ghaf = {
+              profiles = {
+                applications.enable = false;
+                graphics.enable = false;
+                #TODO clean this up when the microvm is updated to latest
+                release.enable = variant == "release";
+                debug.enable = variant == "debug";
+              };
+              development = {
+                debug.tools.enable = variant == "debug";
+                ssh.daemon.enable = true;
+              };
+              windows-launcher.enable = false;
+            };
+            nixpkgs = {
+              localSystem.config = "x86_64-unknown-linux-gnu";
```

Comment:
This should be buildPlatform.system = "x86_64-linux";

Reply:
Done